### PR TITLE
Mr carousel

### PR DESCRIPTION
**Pull request by Maggieryder**
Added Carousel component

Options:
with nav
no nav
fade transtion

### CODE REVIEW

File: packages/addons-website/scss/components/_carousel.scss [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+          &:first-child {
+            opacity: 1;
+          }
+        }
+      }
+    }
+  }  
+}
+
+// .#{$prefix}--carousel-slide-wrapper {
```
Comment (alisonjoseph) [inline packages/addons-website/scss/components/_carousel.scss]: did you want to remove?

File: packages/addons-website/src/components/ImageComponent/ImageComponent-story.js [hunk 2]
```diff
@@ -12,25 +12,26 @@ storiesOf('Image Component', module).add(
...
-          </ImageComponent>,
-    {
-      info: {
-        text: 'Image Component no Zoom',
-      },
-    }
-  )
-  ;
+)
+// .add(
```
Comment (alisonjoseph) [inline packages/addons-website/src/components/ImageComponent/ImageComponent-story.js]: was this supposed to be commented out?

Comment (Maggieryder) [reply packages/addons-website/src/components/ImageComponent/ImageComponent-story.js]: @alisonjoseph this is not a component I was working on but I may have inadvertently commented it out. I have un-commented

**Comment by vpicone** [comment]
Thanks for all your hard work on this stuff @Maggieryder!

**Pull request merged**